File: core/extension/src/main/java/com/rk/extension/ExtensionAPI.java
package com.rk.extension;

public abstract class ExtensionAPI {
    public abstract void onPluginLoaded();
    public abstract void onAppCreated();
    public abstract void onAppLaunched();
    public abstract void onAppPaused();
    public abstract void onAppResumed();
    public abstract void onAppDestroyed();
    public abstract void onLowMemory();
}

File: core/extension/src/main/java/com/rk/extension/ExtensionAPI.kt
package com.rk.extension


abstract class ExtensionAPI {
    abstract fun onPluginLoaded()
    abstract fun onAppCreated()
    abstract fun onAppLaunched()
    abstract fun onAppPaused()
    abstract fun onAppResumed()
    abstract fun onAppDestroyed()
    abstract fun onLowMemory()
}
